docs_src/microservices/application/services/AppRecordReplay/GettingStarted.md

```diff
@@ -0,0 +1,214 @@
+    level=DEBUG ts=2023-10-02T18:22:12.253729459Z app=app-record-replay source=manager.go:651 msg="ARR Event Count: received event to be recorded. Current event count is 4"
+    level=DEBUG ts=2023-10-02T18:22:12.335495342Z app=app-record-replay source=manager.go:651 msg="ARR Event Count: received event to be recorded. Current event count is 5"
+    level=DEBUG ts=2023-10-02T18:22:27.337831668Z app=app-record-replay source=manager.go:651 msg="ARR Event Count: received event to be recorded. Current event count is 6"
+    level=DEBUG ts=2023-10-02T18:22:42.254128347Z app=app-record-replay source=manager.go:673 msg="ARR Process Recorded Data: Recording of Events has ended and functions pipeline has been removed"
+    level=DEBUG ts=2023-10-02T18:22:42.254513348Z app=app-record-replay source=manager.go:696 msg="ARR Process Recorded Data: 6 events in 1m5.942835741s have been saved for replay"
+    ```
+
+!!! warning
+    Since the storage model is simple in-memory storage, restarting the service will result in loss of recorded data. See [Export a Recorded Session](#export-a-recorded-session) section below for details on how to save recoded data for later usage.
+
+#### Check Recording Status
+
+The status of a recording session can be checked while it is running or after is has completed. Review the **Recording Status** GET API in the [API Reference](ApiReference.md) section and use the **Recording Status** request from the PostMan collection referenced above. Press `Send` in PostMan to get the recording status.
+
+!!! example - "Example Recording Status responses"
+    ```json
+    
+    Recording currently running response:
+    {
+        "inProgress": true,
+        "eventCount": 1,
+        "duration": 10057080770
+    }
+    
+    Recording completed response:
+    {
+        "inProgress": false,
+        "eventCount": 6,
+        "duration": 65942835741
+    }
+    ```
+
+#### Cancel Recording 
+
+A recording session can be canceled while it is running. Review the **Cancel Recording** DELETE API in the [API Reference](ApiReference.md) section and use the **Cancel Recording** request from the PostMan collection referenced above. Press `Send` in PostMan to cancel the recording session.
+
+!!! note
+    This API will return a **202 - Accepted** response if the recording can be canceled, otherwise it will return an error such as "***failed to cancel recording: no recording currently running***"
+
+### Replaying a Session
+
+#### Start Replay
+
+To start a replay session first review the **Start Replay** POST API in the [API Reference](ApiReference.md) section and then use the `Start Replay` request from the PostMan collection referenced above. 
+
+Set the `replayRate` to desired value. Value must be greater than zero. Values less than 1 replay slower and values greater than 1 replay faster than originally recorded. Value of 1 replays at the originally recorded rate.
```

Comment:
is there a max replay rate?

Reply:
Nope. They can shoot themselves in the foot by setting it to large number. :-)